RocketApp/Domain/Sources/RocketsClient/RocketsClient+liveReact.swift

```diff
@@ -0,0 +1,33 @@
+import Dependencies
```

Comment:
Nepouzivej slovo `React`, to je strasne pretizeny slovo. Normalne to pojmenuj `Combine`.

Reply:
Prejmenovano.